tools/cldr-code/src/main/java/org/unicode/cldr/util/VoterReportStatus.java

```diff
@@ -10,9 +10,7 @@
      * Enumeration for the reports. In order.
      */
     public enum ReportId {
-        datetime,
-        zones,
-        compact,  // aka 'numbers'
+        datetime, zones, compact, // aka 'numbers'
```

Comment:
supposing we eventually make the naming more consistent, which name is better, "compact" or "numbers"?

Reply:
These are the URLs also. I think what we should really do is think about the whole structure of reports and come up with a new system for them.